extensions/guacamole-auth-http/src/main/java/org/apache/guacamole/auth/http/AuthenticationProviderService.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2015 Glyptodon LLC
```

Comment:
If copying the `AuthenticationProviderService` from something else, should probably copy the Apache-licensed one.